libs/langchain/langchain/output_parsers/format_instructions.py

```diff
@@ -25,3 +25,18 @@
 ```
 {schema}
 ```"""
+
+
+XML_FORMAT_INSTRUCTIONS = """The output should be formatted as a XML file.
+Always mention the encoding of the file in the first line.
```

Comment:
as in other comment, I don't think it is useful to have encoding. Generated tokens are rather well-defined and we should keep output leaner.